### DISCUSSION TITLE
Search on a column that is not displayed

### DISCUSSION THREAD

**Discussion by demeus** [rappasoft/laravel-livewire-tables#1030] [General]
How to do a search on a column that is not displayed
i try this code 
`
            Column::make("content", 'content')
                  ->searchable()
                  ->isHidden(),`

but this code doesn't work either

**Comment by lrljoe** [comment]
Unless I am misunderstanding your question, you can use the filter function to do filtering/searching on columns that aren't displayed.

If I've missed your point please clarify!

**Comment by demeus** [comment]
The fact is that I would not like to use a filter, and write the phrase in the search field

**Reply by lrljoe** [reply]
Use a Builder variant of the component in that case, you can then customise the query to search on non-displayed fields however you like.

**Comment by demeus** [comment]
Ok, Thanks :)

**Comment by lrljoe** [comment]
Let me know if this needs further clarification, otherwise if you can mark it as complete, then that'll be great!
